### PR TITLE
Progress update

### PR DESCRIPTION
**Pull request by HaSistrunk**
Would appreciate a second set of eyes on the text of this post

### CODE REVIEW

File: _posts/2017-08-16-progress.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: default
+img: data-transfer.jpg
+category: Progress
+title: Developing the Transfer Process
+description: |
+---
+  Our recent work has focused on building out the part of Project Electron that facilitates the transfer of records from donors to the Rockefeller Archive Center. Hillel recently posted an [overview of the requirements](http://blog.rockarch.org/?p=1784) that we defined for the structure of these transfers, which are also [shared in more detail on GitHub](https://github.com/RockefellerArchiveCenter/project_electron/tree/master/transfer). We also held a hackathon, hosted by our partners at Marist College IT, to plan and start to build out the infrastructure. Check out Hannah's [blog post](http://blog.rockarch.org/?p=1815) for more details on the hackathon. We are are looking forward to testing transfers very soon!
```
Comment (helrond) [inline _posts/2017-08-16-progress.markdown]: I would say "In addition, we held a hackathon..." to avoid two "also"s in such close proximity.

**Comment by p-galligan** [comment]
The first sentence reads a little clunky to me. Personally, I'd change it to: 

> We've focused our recent work on building out Project Electron's functionality to facilitate the transfer of records from donors.

Take it or leave it though.

**Comment by helrond** [comment]
Or just dispense with the "Project Electron" altogether since its on the project website and say 

>We've focused our recent work on building out functionality to facilitate the transfer of records to the RAC from donors.

**Comment by HaSistrunk** [comment]
Great suggestions, thanks! Incorporated them and going to merge it if no objections

**Review by helrond — APPROVED** [review]

**Pull request merged**